wifimgr.py

```diff
@@ -280,8 +372,7 @@ def start(port=80):
         client, addr = server_socket.accept()
         print('client connected from', addr)
         try:
-            client.settimeout(5.0)
-
+            client.settimeout(15.0)
```

Comment:
can you add a comment here why such a long timeout is needed?

Reply:
It fixed the timeout that some times occured on the not connected page.